### DISCUSSION TITLE
Loving the new fm and vpm oscilators!

### DISCUSSION THREAD

**Discussion by ptowns** [bangcorrupt/hacktribe#49] [Q&A] [answered]
Probably way over my head, but care to share some insight to how you created these? Where you able to pick the action of the edit knob when making these?

**Comment by bangcorrupt** [comment, marked as answer]
These are the same FM and VPM oscillators that are already in the synth, just with different parameters set.  Most of the oscillators only have one controllable parameter, see the [KingKORG Parameter Guide](https://www.korg.com/us/support/download/manual/0/97/1953/) for details.  The oscillator presets allow setting default values for 2 parameters and assigning one of those parameters to the edit knob.  The new FM and VPM oscillators have mod depth assigned to the edit knob and fixed values for frequency that align with semitones.
